compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/SDLFileGenerator.java

```diff
@@ -0,0 +1,391 @@
+    }
+
+    private String createFieldDescription(String description) {
+        if (description == null || description.isEmpty()) {
+            return EMPTY_STRING;
+        } else {
+            List<String> subStrings = new ArrayList<>();
+            for (String subString : description.split(LINE_SEPARATOR)) {
+                subStrings.add(getFormattedString(COMMENT_FORMAT, INDENTATION, subString));
+            }
+            return getFormattedString(DESC_FORMAT, String.join(LINE_SEPARATOR, subStrings));
+        }
+    }
+
+    private String createDirectiveLocation(List<DirectiveLocation> location) {
+        List<String> locations = new ArrayList<>();
+        for (DirectiveLocation directiveLocation : location) {
+            locations.add(directiveLocation.name());
+        }
+        return String.join(PIPE_SIGN, locations);
+    }
+
+    private String createDeprecate(EnumValue enumValue) {
+        if (enumValue.isDeprecated()) {
+            if (enumValue.getDeprecationReason() != null) {
+                return getFormattedString(DEPRECATE_FORMAT, DEPRECATE, enumValue.getDeprecationReason());
+            }
+            return DEPRECATE;
+        }
+        return EMPTY_STRING;
+    }
+
+    private String createDeprecate(Field field) {
+        if (field.isDeprecated()) {
+            if (field.getDeprecationReason() != null) {
+                return getFormattedString(DEPRECATE_FORMAT, DEPRECATE, field.getDeprecationReason());
+            }
+            return DEPRECATE;
+        }
+        return EMPTY_STRING;
+    }
+
+    private Boolean isIntrospectionType(Type type) {
+        for (IntrospectionType value : IntrospectionType.values()) {
+            if (value.getName().equals(type.getName())) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private Boolean isBuiltInScalarType(Type type) {
+        for (ScalarType value : ScalarType.values()) {
+            if (value.getName().equals(type.getName())) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private Boolean isDefaultDirective(Directive directive) {
+        for (DefaultDirective value : DefaultDirective.values()) {
+            if (value.getName().equals(directive.getName())) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private String getFormattedString(String format, String... args) {
+        return String.format(format, (Object[]) args);
+    }
+
+    private void writeFile(String content) {
+        Path filePath = Paths.get(getTargetPath(), getSchemaFileName()).toAbsolutePath();
+        try {
+            createFileIfNotExists(filePath);
+            Files.write(filePath, Collections.singleton(content));
+        } catch (IOException e) {
+            this.stdError.println("WARNING! SDL schema file generation failed. " + e.getMessage());
```

Comment:
Let's keep these messages align with the other warning/error messages. Please check the format and update

```suggestion
            this.stdError.println("[Warning] graphql schema file generation failed: " + e.getMessage());
```